crates/next-core/src/next_app/app_client_references_chunks.rs

```diff
@@ -53,76 +53,92 @@ pub async fn get_app_client_references_chunks(
-                                    if let Some(ssr_chunking_context) = ssr_chunking_context {
-                                        let ssr_chunk_group = ssr_chunking_context
-                                            .root_chunk_group(Vc::upcast(
-                                                ecmascript_client_reference_ref.ssr_module,
+                                        (
+                                            client_chunk_group.assets,
+                                            client_chunk_group.availability_info,
+                                        ),
+                                        if let Some(ssr_chunking_context) = ssr_chunking_context {
+                                            let ssr_chunk_group = ssr_chunking_context
+                                                .root_chunk_group(Vc::upcast(
+                                                    ecmascript_client_reference_ref.ssr_module,
+                                                ))
+                                                .await?;
+
+                                            Some((
+                                                ssr_chunk_group.assets,
+                                                ssr_chunk_group.availability_info,
                                             ))
-                                            .await?;
+                                        } else {
+                                            None
+                                        },
+                                        if let Some(ssr_chunking_context) = ssr_chunking_context {
+                                            let ssr_chunk_group = ssr_chunking_context
+                                                .root_chunk_group(Vc::upcast(parent_module))
+                                                .await?;
 
-                                        Some((
-                                            ssr_chunk_group.assets,
-                                            ssr_chunk_group.availability_info,
-                                        ))
-                                    } else {
-                                        None
-                                    },
-                                )
-                            }
-                            ClientReferenceType::CssClientReference(css_module) => {
-                                let client_chunk_group = client_chunking_context
-                                    .root_chunk_group(Vc::upcast(css_module))
-                                    .await?;
+                                            Some((
+                                                ssr_chunk_group.assets,
+                                                ssr_chunk_group.availability_info,
+                                            ))
+                                        } else {
+                                            None
+                                        },
```

Comment:
unused? can be removed?

Reply:
When removed you don't need `parent_module` anymore and the other code can be removed too